common/src/lazy_pages.rs

```diff
@@ -23,48 +23,47 @@ use gear_core::{
     ids::ProgramId,
     memory::{HostPointer, Memory, PageBuf, PageNumber},
 };
-use gear_runtime_interface::{gear_ri, GetReleasedPageError, MprotectError};
+use gear_runtime_interface::{gear_ri, RIError};
 use sp_std::{
     collections::{btree_map::BTreeMap, btree_set::BTreeSet},
     vec::Vec,
 };
 
 #[derive(Debug, Clone, PartialEq, Eq, derive_more::Display)]
 pub enum Error {
-    #[display(fmt = "{}", _0)]
-    Mprotect(MprotectError),
-    #[display(fmt = "{}", _0)]
-    GetReleasedPage(GetReleasedPageError),
-    #[display(fmt = "Cannot convert vec to page data")]
-    VecToPageData,
-    #[display(fmt = "Cannot find page data in storage")]
-    NoPageDataInStorage,
-    #[display(fmt = "Released page {:?} has initial data", _0)]
-    ReleasedPageHasData(PageNumber),
+    #[display(fmt = "RUNTIME INTERFACE ERROR: {}", _0)]
```

Comment:
I'm not sure how to do that, but maybe there are ways to make the  "RUNTIME INTERFACE ERROR" sub-string common for all variants of errors? The one that comes up to my mind is to make it the next way: you separate in this module `Error` and `RuntimeErrorInner`. The latter one has all the variants of the current `Error` version. The new `Error` version has only one - `RIError(RuntimeErrorInner)`, for which the next attribute will be defined: `#[display(fmt = "RUNTIME INTERFACE ERROR: {:?}", _0)]. Maybe there are some easy ways. dunno.

Reply:
I agree with ISab

Reply:
I don't see any reasons to do this . What the reason to make one more Error type? I've just marked errors which I want to mark that the problem is connected with using RI.

Reply:
resolved after discussion